src/codegeneration/ClassTransformer.js

```diff
@@ -107,15 +114,40 @@ function classCall(func, object, staticObject, superClass) {
       `($traceurRuntime.createClass)(${func}, ${object}, ${staticObject})`;
 }
 
+function methodNameFromToken(token) {
```

Comment:
what should I call it then? frankly, I don't fully understand all the different tokens and not tokens;-)